katas/content/phase_estimation/state_eigenvector/index.md

```diff
@@ -0,0 +1,12 @@
+**Inputs:**
+
+1. A single-qubit unitary $U$.
+2. A single-qubit state $\ket{\psi}$, given as a unitary $P$ that prepares it from the $\ket{0}$ state. In other words, the result of applying the unitary $P$ to the state $\ket{0}$ is the $\ket{\psi}$ state:
+   $$P\ket{0} = \ket{\psi}$$
+
+**Output:** Return true if the given state is an eigenstate of the given unitary, and false otherwise.
+
+<details>
+  <summary><b>Need a hint?</b></summary>
+  The library operation <code>CheckZero</code> allows you to check whether the state of the given qubit is $\ket{0}$.
+</details>
```

Comment:
There needs to be some extra spacing here, otherwise the parser gets confused and doesn't render the $\ket{0}$ correctly.
```suggestion
<details>
  <summary><b>Need a hint?</b></summary>

The library operation <code>CheckZero</code> allows you to check whether the state of the given qubit is $\ket{0}$.

</details>
```